dolweb/downloads/templates/downloads-index.html

```diff
@@ -81,10 +80,11 @@ <h1>{% trans "Development versions" %}</h1>
 <div id="download-stable">
 <h1>{% trans "Stable versions" %}</h2>
 
-<div class="alert alert-info">{% blocktrans %}
-    Stable versions are released after a lot of testing to ensure emulation
-    performance and stability. However, since they are released less often,
-    they might be outdated and lacking some new features.
+<div class="alert alert-danger">{% blocktrans %}
```

Comment:
alert-danger seems a bit overkill. Maybe warning or info?

Reply:
I suspect if we went with info too many people would just skim over it, but the color contrast of warning should be enough.